src/share/import.js

```diff
@@ -0,0 +1,57 @@
+import fs from '../lib/fs';
+import Buffer from '../lib/buffer';
+import Path from '../lib/path';
+import WebTorrent from 'webtorrent';
+const sh = new fs.Shell();
+
+document.getElementById('btnDownload').addEventListener('click', startDownload);
+console.log('inside of import!');
+
+const downloadAllTorrentFiles = torrent =>
+    Promise.all(
+        torrent.files.map(
+            file =>
+                new Promise((resolve, reject) => {
+                    var filename = Path.join('/', file.path);
+                    file.getBuffer((err, buffer) => {
+                        if (err) {
+                            return reject(err);
+                        }
+                        fs.writeFile(filename, new Buffer(buffer), err => {
+                            if (err) {
+                                console.error(
+                                    'Error writing file, error was ',
+                                    err,
+                                    filename
+                                );
+                                return reject(err);
+                            } else {
+                                console.log('Wrote to ', filename);
+                                resolve();
+                            }
+                        });
+                    });
+                })
+        )
+    );
+
+function startDownload() {
+    let magnetURI = document.getElementById('torrentId').value;
+
+    if (!magnetURI) {
+        console.log('Need a torrent magnet link to proceed.');

```

Comment:
Console logs are great for debugging, but a user won't see them.  You need to surface this info to the user.  One way to do it is to have `<div id="error-info" hidden>You need to provide a magnet URI to download</div>` and then remove the `hidden` attribute when you want to show it, or use a `.hidden` class you toggle.